pep-0517.txt

```diff
@@ -166,48 +166,71 @@ The build backend object is expected to have attributes which provide
 some or all of the following hooks. The common ``config_settings``
 argument is described after the individual hooks::
 
-  def get_build_requires(config_settings):
+  def build_sdist(source_directory, sdist_directory, config_settings):
+      ...
+
+MUST build an unpacked sdist from the ``source_directory`` and place it into the
```

Comment:
Do we really want to leave it to the backend to decide what goes into a sdist? Certainly the backend needs to say "we must have these files if we are to be able to build a wheel", but for other things (like allowing the user to specify additional files to include in the sdist, like LICENSE) it seems like it would be better to have a standard method, probably specified via ```pyproject.toml``` that sdist-building tools would deal with, rather than expecting backends to all duplicate the same logic.

I think I'd prefer a ```get_build_files``` hook that has to return a list of the files needed for a build. Backends SHOULD return a minimal list. The files returned MUST be appropriate for including in a sdist (I think it's OK to leave this use of "sdist" intentionally vague), so VCS control files, tox caches, etc, need to be excluded.